### PR TITLE
HZC-1196 - Update support.adoc

### PR DESCRIPTION
**Pull request by fantkolja**
https://hazelcast.atlassian.net/browse/HZC-1196

### CODE REVIEW

File: docs/modules/ROOT/pages/support.adoc [hunk 1]
```diff
@@ -1,33 +1,38 @@
...
-*SLA:* First Response < 1 Business Day
+
+*SLA:*
+
+First Response < 1 Business Day
 
 == Enterprise
 
-For enterprise support please email to mailto:sales@hazelcast.com[]. 
+For enterprise support please email to mailto:sales@hazelcast.com[].
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/support.adoc:20]:
```suggestion
If you're interested in Enterprise support, contact us at mailto:sales@hazelcast.com[].
```

File: docs/modules/ROOT/pages/support.adoc [hunk 2]
```diff
@@ -1,33 +1,38 @@
...
 == Standard
 
 Standard support is available for all users with non-free clusters.
+
 *Coverage:* 24 x 7 x 365
+
 *Support Channel:* link:{url-zendesk-hazelcast}[ZenDesk]
-*SLA:* First Response < 1 Business Day
+
+*SLA:*
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/support.adoc:14]:
```suggestion
*Our Promise to Respond:*
```

SLA doesn't seem very friendly. If we would rather use SLA, we should write it in full for anyone who does not know what that means (Service-Level Agreement).

File: docs/modules/ROOT/pages/support.adoc [hunk 3]
```diff
@@ -1,33 +1,38 @@
...
+
+First Response < 1 Business Day
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/support.adoc:16]:
```suggestion
We will respond in less than one business day.
```

File: docs/modules/ROOT/pages/support.adoc [hunk 4]
```diff
@@ -1,33 +1,38 @@
...
-For enterprise support please email to mailto:sales@hazelcast.com[]. 
+For enterprise support please email to mailto:sales@hazelcast.com[].
+
 *Coverage:* 24 x 7 x 365
-*Support channels: * link:{url-zendesk-hazelcast}[ZenDesk], Emergency Telephone Number
-*Emergency Patches*
+
+*Support channels:* link:{url-zendesk-hazelcast}[ZenDesk], Emergency Telephone Number
+
 *SLA*
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/support.adoc:26]:
```suggestion
*Our Promise to Respond:*
```

File: docs/modules/ROOT/pages/support.adoc [hunk 5]
```diff
@@ -1,33 +1,38 @@
...
+
 Severity 1: First Response < 30mins (24 x 7 x 365)
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/support.adoc:28]: Perhaps we could give users an indication of what we mean by these severity levels?

Also why do we repeat '(24 x 7 x 365)'? We have already mentioned that under 'Coverage'.

File: docs/modules/ROOT/pages/support.adoc [hunk 6]
```diff
@@ -1,33 +1,38 @@
...
-== Community
-
-Community support is available for free tier users. 
-*Support Channel:* link:{url-slack-hazelcast}[Slack]
-
 == Standard
 
 Standard support is available for all users with non-free clusters.
+
 *Coverage:* 24 x 7 x 365
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/support.adoc:10]:
```suggestion
*Coverage:* 24 hours a day, 7 days a week, 365 days a year
```

**Review by JakeSCahill — COMMENTED** [review]
Seeing as we are updating this doc, I just left some general comments.

File: docs/modules/ROOT/pages/support.adoc (hunk 5 above)
Comment (fantkolja) [reply docs/modules/ROOT/pages/support.adoc]: Could you please check the indications added?

File: docs/modules/ROOT/pages/support.adoc [hunk 7]
```diff
@@ -1,33 +1,38 @@
...
+
+*Support channels:* link:{url-zendesk-hazelcast}[ZenDesk], Emergency Telephone Number
+
+*Our Response Times:*
+
+Severity 1 (a critical incident with very high impact): First Response < 30mins
+
+Severity 2 (a major incident with significant impact): First Response < 2 Hours
+
+Severity 3 (a minor incident with low impact): First Response < 4 Hours
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/support.adoc:32]:
```suggestion
Severity 3 (low impact on applications): We will respond in less than 4 hours.
```

File: docs/modules/ROOT/pages/support.adoc [hunk 8]
```diff
@@ -1,33 +1,38 @@
...
+
+Severity 4 (an insignificant issue with little to no impact): First Response < 1 Business Day
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/support.adoc:34]:
```suggestion
Severity 4 (insignificant issue with little to no impact): We will respond in less than one business day.
```

File: docs/modules/ROOT/pages/support.adoc (hunk 5 above)
Comment (JakeSCahill) [reply docs/modules/ROOT/pages/support.adoc]: Looks good, thanks. I just added some more context because high, medium, and low impact is ambiguous.

Comment (fantkolja) [reply docs/modules/ROOT/pages/support.adoc]: All accepted, thanks a lot @JakeSCahill !!!

**Review by JakeSCahill — APPROVED** [review]
LGTM 👌

**Review by horakihor — APPROVED** [review]
LGTM

**Pull request merged**